package.json

```diff
@@ -26,9 +28,13 @@
     "@vitejs/plugin-vue": "^3.2.0",
     "@vue/test-utils": "^2.2.4",
     "autoprefixer": "^10.4.11",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-vue": "^9.20.1",
```

Comment:
can you move this dependecies to dev dependecies?

Reply:
Yess, you are right, it was avoided from my attention

Reply:
sorry, github "smart" decisions on hiding some content made it look like they were under dependecies. looks good, thank you!! 🙏